website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,15 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Add recipient and sender parameters for individual name parts
+Starting 2025-XX-XX, the new name-related parameters `recipient[first_name]`,
+`recipient[middle_name]`, `recipient[last_name]`, `sender[first_name]`,
+`sender[middle_name]` and `sender[last_name]` are supported.  
+This will in time push for the deprecation of `card[name]`, `recipient[name]` 
```

Comment:
```suggestion
`sender[middle_name]` and `sender[last_name]` are supported.
This will in time push for the deprecation of `card[name]`, `recipient[name]`
```